### PR TITLE
SEARCH-1526 - Change logger implementation

### PR DESCRIPTION
**Pull request by keerthi16**
## Description
Change logger implementation
[SEARCH-1526](https://perzoinc.atlassian.net/browse/SEARCH-1526)

## Solution Approach
Changes logger implementation to match with electron

## Documentation
N/A

## Related PRs
List related PRs against other branches / repositories:

branch | PR
------ | ------
SymphonyElectron `master` | [#676](https://github.com/symphonyoss/SymphonyElectron/pull/676)
SymphonyElectron  `5.x` | [#677](https://github.com/symphonyoss/SymphonyElectron/pull/677)

## QA Checklist
- [x] Unit-Tests

### CODE REVIEW

File: src/compressionLib/compression.ts [hunk 1]
```diff
@@ -22,25 +21,25 @@ async function compression(pathToFolder: string, outputPath: string, callback: (
         try {
             const { stdout, stderr } = await exec(`cd "${ROOT_PATH}" && tar cf - "${pathToFolder}" | "${searchConfig.LIBRARY_CONSTANTS.MAC_LIBRARY_FOLDER}/lz4.exec" > "${outputPath}.tar.lz4"`);
             if (stderr) {
-                log.send(logLevels.INFO, `compression stderr: ${stderr}`);
+                logger.info(`compression: compression stderr: ${stderr}`);
```
Comment (KiranNiranjan) [inline src/compressionLib/compression.ts]:
```suggestion
                logger.info(`compression: compression stderr:`,  stderr);
```

All the concatenated values can be passed as the second arg

File: src/search.ts [hunk 2]
```diff
@@ -108,11 +107,11 @@ export default class Search extends SearchUtils implements SearchInterface {
 
                 clearSearchData.call(this);
                 if (res === undefined || res === null || res < 0) {
-                    log.send(logLevels.ERROR, 'Deserialization of Main Index Failed-> ' + error);
+                    logger.error(`search: Deserialization of Main Index Failed-> ${error}`);
```
Comment (KiranNiranjan) [inline src/search.ts]: Same as above for error logs

File: src/compressionLib/compression.ts (hunk 1 above)
Comment (keerthi16) [reply src/compressionLib/compression.ts]: Done 👍

File: src/search.ts (hunk 2 above)
Comment (keerthi16) [reply src/search.ts]: Done 👍

**Comment by keerthi16** [comment]
Published @ `swift-search@1.55.2-beta.2`
Screenshot 2019-06-12 at 6 51 22 PM

**Pull request merged**